### ISSUE TITLE
Add training images

### ISSUE THREAD

**Issue by raialvaro** [gabrieltseng/solar-panel-segmentation#9]
Hi again @gabrieltseng 

I have to admit that your model works fine. It's amazing!

I would like to know how I can take advantage of the model and train it with new images.

I have observed in his way of training and in the metados I see that he has the coordinates of the pixels of the polygons, vertices, etc.

![image](https://user-images.githubusercontent.com/71493478/94525939-16902600-0235-11eb-9fcd-047c4ad23b79.png)

Could you explain to me what data I would need from the new images I want to add to pre-train the model?

so, what step would I have to do?

Would be appreciated

Thank you very much

**Comment by gabrieltseng** [comment]
Hi @raialvaro ,

To pretrain the model with new images, you will need to get the new images into the same format as what's described [here](https://www.nature.com/articles/sdata2016106). 

If you want to use the original data as well when you train the model, you will need to get images of similar resolution. If not, you can use other sources (e.g. Sentinel-2) to train the models, but the current data might not be usable due to resolution difference.

**Comment by raialvaro** [comment]
Hi @gabrieltseng 

Thank so much for the info!

I want to train with PNOA images that have 25 cm resolution

Could you load the model and train with the new images PNOA and save time, or is it best to retrain with all the images from Fresno, Oxnard, Stockton and the new images?

I prefer to load the pretrained model and train it with the new images to save computing time. But perhaps it is not recommended.

What is your opinion??

Thanks so much!
